payments/index.md

```diff
@@ -66,6 +65,24 @@ POST https://b.paylike.io/payments
     returnUrl: URL, // optional, return to this URL for redirect-based flows
   },
 
+  // optional
+  source: {
+    holder: 'destination', // optional
+  }
+  // optional
+  destination: {
+    id: String, // optional, length: 1..1024
+    // optional
+    holder: {
+      id: String, // optional, length: 1..1024
+      name: String, // optional, length: 1..1024
+      address: String, // optional, length: 1..1024
+      city: String, // optional, length: 1..1024
+      state: String, // optional, length: 1..1024
```

Comment:
Maybe at least the `state` (as is the country) should be expected as `{2,2}`?

Reply:
That sounds reasonable..

Reply:
So, I changed it to be an actual ISO code instead:

> The second part is a string of up to three alphanumeric characters